DuckDuckGo/ContentBlockerSettingsChangeDelegate.swift

```diff
@@ -21,6 +21,4 @@ import Foundation
 
 protocol ContentBlockerSettingsChangeDelegate: class {
 
-    func contentBlockerSettingsDidChange()
-
 }
```

Comment:
Since this delegate seems to not be used at all, it should be safe to remove it.